docs/tutorials/tutorial-hdfs-logs-distributed-search-aws-s3.md

```diff
@@ -45,116 +46,152 @@ Let's create an index configured to receive these logs.
 
 ```bash
 # First, download the hdfs logs config from Quickwit repository.
-curl -o hdfslogs_index_config.json https://raw.githubusercontent.com/quickwit-inc/quickwit/main/examples/index_configs/hdfslogs_index_config.json
+curl -o hdfslogs_index_config.yaml https://raw.githubusercontent.com/quickwit-inc/quickwit/main/config/tutorials/hdfs-logs/index-config.yaml
 ```
 
-The index config defines 5 fields: `timestamp`, `severity_text`, `body`, and two object fields
-for the nested values `resource` and `attributes`. 
-It also sets the [default search field](../reference/index-config.md) and a timestamp field. 
-This timestamp field will be used by Quickwit for [splits pruning](../overview/architecture.md) at query time to boost search speed. Check out the [index config docs](../reference/index-config.md) for details.
+The index config defines four fields: `timestamp`, `severity_text`, `body`, and one object field
+for the nested values `resource.service` . It also sets the `default_search_fields`, the `tag_fields`, and the `timestamp_field`. The `timestamp_field` and `tag_fields` are used by Quickwit for [splits pruning](../overview/architecture.md) at query time to boost search speed. Check out the [index config docs](../reference/index-config.md) for more details.
+
+```yaml title="hdfslogs_index_config.yaml"
+version: 0
+
+doc_mapping:
+  field_mappings:
+    - name: timestamp
+      type: i64
+      fast: true
+    - name: severity_text
+      type: text
+      tokenizer: raw
+    - name: body
+      type: text
+      tokenizer: default
+      record: position
+    - name: resource
+      type: object
+      field_mappings:
+        - name: service
+          type: text
+          tokenizer: raw
+  tag_fields: [tenant_id]
+  store_source: true
+
+indexing_settings:
+  timestamp_field: timestamp
+
+search_settings:
+  default_search_fields: [severity_text, body]
+```
 
+To index our dataset and serve it on a Quickwit cluster, we first need to create a few search nodes configuration files. Please, define your S3 bucket path now to ease the config files creation process.
```

Comment:
I think it's important to do things step by step for the new user.
Here there is something confusing: we are defining searcher's conf files but we want to first index. 

I think the good way to go is to say: let's first create and ingest our index on S3. For that we need to define a quickwit config file. We should not talk about searchers immediately.

Once we have indexed it then we can think about how to set up a search cluster.

The tutorial was about configuring an cluster on AWS but your configuration will not work with `rest_listen_address: 127.0.0.1` and the different ports used here. I added these config files for a local search cluster :)

If we are on several machines, we can even think about having only one config file named `quickwit.yaml` that sets the right IPs directly.